### ISSUE TITLE
Taxes collected in ETH

### ISSUE THREAD

**Issue by codynhat** [Geo-Web-Project/specs#8]
Taxes are collected and stored in ETH. This will require a ETH/USD price oracle integration.

**Comment by codynhat** [comment]
With the strategy we've discussed around collecting taxes based on an expiration date, accepting other non-stable currencies would add additional work. We can look into supporting this eventually, but perhaps for the MVP we just stick to everything being done in DAI. Thoughts? @gravenp

**Comment by gravenp** [comment]
Yeah, I started going down the road of all the complications surrounding return/assumption of prepaid taxes with non-stable coin and definitely seemed like a lot of extra work. For MVP, keeping it simple seems like the best course. Dai is relatively stable, but does have a trading band. USDC always trades for $1 right? Would that make it easier to transition to USD denominated land parcels with many options for payment down the road?

**Comment by codynhat** [comment]
Are we settling on just using DAI everywhere? Both for collecting fees and setting prices?

We discussed DAI not being exactly pegged to USD, and how explaining what "DAI" is to non-web3 users may be weird. Is this a concern for the MVP since our target is initially going to be Web3 users who understand what DAI is?

**Comment by gravenp** [comment]
Yeah, let's start with DAI for both fees and prices. Like you said, appropriate for the target initial adopters. We can explore additional options in the step up from MVP.
